src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -13,6 +13,7 @@
 
 using Xunit;
 using Xunit.Abstractions;
+using System.Runtime.InteropServices.GeneratedMarshalling;
```

Comment:
why this is needed? (I did nots ee anything obvious)

Reply:
typo in code made VS automatically add this using :/